rhombus/scribblings/ref-io.scrbl

```diff
@@ -4,15 +4,23 @@
 
 @title{Input and Output}
 
+A @deftech{port} is an input or output stream for a file, network
+connection, terminal, etc. An @deftech{input port} is specifically for
+output, while an @deftech{output port} is specifically for input.
+
+@dispatch_table(
+  "output port"
+  @rhombus(Port.Output)
+  [out.flush(), Port.Output.flush(out)]
+)
+
 @doc(
   annot.macro 'Port'
   annot.macro 'Port.Input'
   annot.macro 'Port.Output'
 ){
 
- The @rhombus(Port, ~annot) annotation is satisified by a
- @deftech{port}, which is an input or output stream for files, network
- connects, terminals, etc. The @rhombus(Port.Input, ~annot) annotation
+ The @rhombus(Port, ~annot) annotation is satisified by a @tech{port}. The @rhombus(Port.Input, ~annot) annotation
  recognizes input ports specifically, while @rhombus(Port.Output, ~annot)
  ports, and it is possible for a port to be both.
```

Comment:
`while @rhombus(Port.Output, ~annot) [recognizes output] ports`?  Also can you add a line break to the first line?

Reply:
HF